Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp

```diff
@@ -343,53 +363,58 @@ void APlayCppSdkActor::SignEip155Transaction(
   });
 }
 
-void APlayCppSdkActor::sendEvent(const FWalletConnectSessionInfo &info) {
+void APlayCppSdkActor::sendEvent(FWalletConnectSessionInfo info) const {
 
   AsyncTask(ENamedThreads::GameThread, [this, info]() {
     this->OnReceiveWalletconnectSessionInfoDelegate.ExecuteIfBound(info);
   });
 }
 
-void UserWalletConnectCallback::onConnected(
+void UserWalletConnectCallback::onConnected( // NOLINT : flase positive, virtual
+                                             // function cannot be static
     const WalletConnectSessionInfo &sessioninfo) const {
   UE_LOG(LogTemp, Log, TEXT("user c++ onConnected"));
   if (APlayCppSdkActor::getInstance()) {
-    FWalletConnectSessionInfo output;
-    convertSession(sessioninfo, output);
-    output.sessionstate = EWalletconnectSessionState::StateConnected;
+    FWalletConnectSessionInfo
+        output = // NOLINT : by unreal engine macro, can be ignored
+        convertSession(sessioninfo, EWalletconnectSessionState::StateConnected);
     APlayCppSdkActor::getInstance()->sendEvent(output);
   }
 }
-void UserWalletConnectCallback::onDisconnected(
+void UserWalletConnectCallback::onDisconnected( // NOLINT : flase positive,
+                                                // virtual function cannot be
+                                                // static
     const WalletConnectSessionInfo &sessioninfo) const {
   UE_LOG(LogTemp, Log, TEXT("user c++ onDisconnected"));
   if (APlayCppSdkActor::getInstance()) {
-    FWalletConnectSessionInfo output;
-    convertSession(sessioninfo, output);
-    output.sessionstate = EWalletconnectSessionState::StateDisconnected;
+    FWalletConnectSessionInfo output = // NOLINT
+        convertSession( // NOLINT : by unreal engine macro, can be ignored
+            sessioninfo, EWalletconnectSessionState::StateDisconnected);
 
     APlayCppSdkActor::getInstance()->sendEvent(output);
   }
 }
-void UserWalletConnectCallback::onConnecting(
-    const WalletConnectSessionInfo &sessioninfo) const {
+void UserWalletConnectCallback::
+    onConnecting( // NOLINT : flase positive, virtual function cannot be static
```

Comment:
The inline comment breaks the format of the function which makes the reading difficult (because formated by clang-format), try put the comment other places. Elsewhere as well.

Reply:
this is for diabling c++ lint, need to be in the line

Reply:
How about fix lints (maybe in another PR), ignore the lint errors at this moment? Putting NOLINT there is not ideal and messy.